docs/static/search.js

```diff
@@ -168,6 +168,12 @@ function initSearch() {
       $searchResultsItems.appendChild(item);
     }
   }, 150));
+
+  window.addEventListener('click', function(e) {
+    if ($searchResults.style.display == "block" && !$searchResults.contains(e.target)) {
+      $searchResults.style.display = "none";
+    }
```

Comment:
You need to handle clicking in the search results div should make it `block` again

Reply:
How you can click `search results div` while it is "none" ?

I tested couple of times. it works fine. if you click outside, the `search results div` will be set to `none`, if you click on searchbox and type something again, `search results div` will be set to "block" and will be shown.

Reply:
check this line https://github.com/getzola/zola/blob/4f023d8c7037a01c1b34136ea204acf84d319f59/docs/static/search.js#L152

Reply:
@Keats